errors thrown in 'make doc'

Atomvm.6.6 is built and working, but I don't see any documentation for the Elixir libraries. In the hope that I could generate them I followed the instructions for sphinx installation and when running in that virtual environment make doc fails with the following output:

> (sphinx) gratitude: make doc
> [  0%] Preparing edown escript
> ===> Verifying dependencies...
> ===> Analyzing applications...
> ===> Compiling edown_dep
> [  0%] Built target edown-escript
> [  0%] Generating edoc markdown for etest
> Generated documentation for library "etest" using source files ["src/etest.erl"] into /home/rashid/Dev/External/AtomVM/AtomVM-0.6.6/build/doc/src/apidocs/erlang/etest
> Fixing links in etest documentation.
> [  0%] Built target edoc-etest
> [  0%] Generating edoc markdown for estdlib
> edoc: error in layout 'edown_layout': {'EXIT',
>                                  {unknown_tag,
>                                   {'#cdata#',
>                                    ["              > apply(fun(R) -> lists:reverse(R) end, [[a, b, c]]).\n           [c,b,a]\n           > apply(fun erlang:atom_to_list/1, ['AtomVM']).\n           \"AtomVM\""]}}}.
> edoc: error in doclet 'edown_doclet': {'EXIT',error}.
> escript: exception exit: error
>   in function  edoc_lib:run_plugin/5 (edoc_lib.erl, line 891)
>   in call from gendoc:main/1 (gendoc.erl, line 31)
>   in call from escript:run/2 (escript.erl, line 904)
>   in call from escript:start/1 (escript.erl, line 418)
>   in call from init:start_it/1
>   in call from init:start_em/1
>   in call from init:do_boot/3
> make[3]: *** [doc/CMakeFiles/edoc-estdlib.dir/build.make:71: doc/CMakeFiles/edoc-estdlib] Error 127
> make[2]: *** [CMakeFiles/Makefile2:7785: doc/CMakeFiles/edoc-estdlib.dir/all] Error 2
> make[1]: *** [CMakeFiles/Makefile2:8036: doc/CMakeFiles/doc.dir/rule] Error 2
> make: *** [Makefile:2730: doc] Error 2
> (sphinx) gratitude:
> 

I'm not at all familiar with how this all works,  I'm running on:

elixir: 1.17.3-otp-27 
erlang: 27.3.4.1
uname -a: Linux gratitude 6.15.6-1-default #1 SMP PREEMPT_DYNAMIC Fri Jul 11 05:06:35 UTC 2025 (a69e09e) x86_64 x86_64 x86_64 GNU/Linux

--- a/.github/workflows/build-docs.yaml
+++ b/.github/workflows/build-docs.yaml
@@ -60,8 +60,7 @@ jobs:
         os: [ ubuntu-24.04 ]
     # The type of runner that the job will run on
     runs-on: ${{ matrix.os }}
-    # Documentation currently fails to build with OTP-27 and recent OTP-26.
-    container: erlang:26.0.2
+    container: erlang:28.1
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:

--- a/.github/workflows/publish-docs.yaml
+++ b/.github/workflows/publish-docs.yaml
@@ -48,7 +48,7 @@ jobs:
         os: [ ubuntu-24.04 ]
     # The type of runner that the job will run on
     runs-on: ${{ matrix.os }}
-    container: erlang:26.0.2
+    container: erlang:28.1
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:

